corehq/blobs/models.py

```diff
@@ -4,3 +4,17 @@
 class BlobMigrationState(models.Model):
     slug = models.CharField(max_length=20, unique=True)
     timestamp = models.DateTimeField(auto_now=True)
+
+
+class BlobExpiration(models.Model):
+    '''
+    This models records when temporary blobs should be deleted
+    '''
+    bucket = models.CharField(max_length=255)
+    identifier = models.CharField(max_length=255)
+    expires_on = models.DateTimeField(db_index=True)
+    created_on = models.DateTimeField(auto_now=True)
+    length = models.IntegerField()
```

Comment:
ah, that certainly could be useful 👍

Incidentally, short of looking at used disk space, I do not know of a way to query the size of all objects stored in Riak, not to mention subsets of that data.